packages/colorpickers/src/styled/ColorDialog/StyledButton.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { Button } from '@zendeskgarden/react-buttons';
+
+const COMPONENT_ID = 'colorpickers.colordialog_button';
+
+const colorStyles = (props: ThemeProps<DefaultTheme>) => {
```

Comment:
This button was changed from @steue's designs. It should default to a `neutralHue` `border-color` (the remaining hover/active/focus stateful colors work as expected) in order to fit in well with surrounding form fields. The `:disabled` `border-color` also needs correct styling.